tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/GivenUsingSuite.scala

```diff
@@ -485,31 +446,6 @@ class GivenUsingSuite extends BaseDottySuite {
     )
```

Comment:
could you add test:
```
scala> given intM: Ord[Int] with Eq[Int] with {
     |   def fx = 3
     |}
// defined object intM
```
Didn't check if it is handled, but seems like a tricky case

Reply:
That might have never actually worked :thinking:

Reply:
Ok, so that was much more difficult than aniticipated - I had to rework given defs, there is no need for separate `decltype` anymore, but we can just use the inits from the Template.